docs/make.jl

```diff
@@ -1,27 +1,34 @@
+using Pkg
+
+# targeting the correct source code
+# this asumes the make.jl script is located in QEDcore.jl/docs
+project_path = Base.Filesystem.joinpath(Base.Filesystem.dirname(Base.source_path()), "..")
+Pkg.develop(; path=project_path)
+
 using LorentzVectorBase
 using Documenter
 
 DocMeta.setdocmeta!(
   LorentzVectorBase, :DocTestSetup, :(using LorentzVectorBase); recursive=true
 )
 
-const numbered_pages = [
-  file for file in readdir(joinpath(@__DIR__, "src")) if
-  file != "index.md" && splitext(file)[2] == ".md"
+pages = [
+  "Home" => "index.md", "Interface" => "10-interface.md", "Reference" => "95-reference.md"
 ]
 
 makedocs(;
   modules=[LorentzVectorBase],
   authors="Uwe Hernandez Acosta <u.hernandez@hzdr.de>",
-  repo="https://github.com/JuliaHEP/LorentzVectorBase.jl/blob/{commit}{path}#{line}",
+  repo=Documenter.Remotes.GitHub("JuliaHEP", "LorentzVectorBase.jl"),
   sitename="LorentzVectorBase.jl",
   format=Documenter.HTML(;
+    prettyurls=get(ENV, "CI", "false") == "true",
     canonical="https://JuliaHEP.github.io/LorentzVectorBase.jl",
-    repolink="https://github.com/JuliaHEP/LorentzVectorBase.jl",
```

Comment:
IIRC,
`repolink` is needed to get to repository when clicking to GitLab, and I think the Documenter complains when it is not specified